iree/compiler/Conversion/CodegenUtils/MarkerUtils.cpp

```diff
@@ -33,6 +33,8 @@ StringRef getWorkgroupMarker() { return "workgroup"; }
 
 StringRef getWorkgroupL1TileMarker() { return "workgroup_l1_tile"; }
 
+StringRef getPromotedWorkgroupTileMarker() { return "promoted_workgroup_tile"; }
```

Comment:
The `getWorkgroupMemoryMarker` is meant for this already. Could just use that.

Reply:
Done!